cli/standalone/serialization.rs

```diff
@@ -202,10 +238,32 @@ impl RemoteModulesStoreBuilder {
       builder.append_le(to.len() as u32);
       builder.append(to);
     }
-    for (media_type, data) in &self.data {
+    builder.append_le(
+      self
+        .data
+        .iter()
+        .map(|(_, data, maybe_transpiled)| {
+          1 + 4
+            + (data.len() as u64)
+            + 1
+            + match maybe_transpiled {
+              Some(transpiled) => 4 + (transpiled.len() as u64),
+              None => 0,
+            }
+        })
+        .sum::<u64>(),
+    );
```

Comment:
This serialization is now quite complex, I'm starting to feel we should add some unit tests for it

Reply:
Also have you considered using `serde_bincode` instead of hand rolling it?

Reply:
I think it's ok. If there were any issues with it then the integration tests would fail.

I think this is more clear what's going on and allows for lazily deserializing certain parts.